langchain/chains/moderation.py

```diff
@@ -37,10 +37,10 @@ class OpenAIModerationChain(Chain):
     @root_validator()
     def validate_environment(cls, values: Dict) -> Dict:
         """Validate that api key and python package exists in environment."""
-        openai_api_key = get_from_dict_or_env(
+        cls.openai_api_key = get_from_dict_or_env(
```

Comment:
imo probably nicer to set in `values` rather than cls (is just more consistent with how we do elsewhere)

unless theres some reason not to?